spec/lib/activity_stream_reader_spec.rb

```diff
@@ -80,11 +81,16 @@
       expect(ActivityStreamLog.last.object_count).to eq 4000
     end
 
-    # it "can call for updates" do
-    #   expect(ActivityStreamLog.count).to eq 1
-    #   described_class.update
-    #   expect(ActivityStreamLog.count).to eq 2
-    # end
+    it "can process the partial activity stream if there is a previous successful run" do
+      asl_old_success
+      expect(ActivityStreamLog.count).to eq 1
+      expect(ActivityStreamLog.last.object_count).to eq asl_old_success.object_count
+      asr.process_activity_stream
+      expect(ActivityStreamLog.count).to eq 2
+      # should be more than just one page of objects
+      expect(ActivityStreamLog.last.object_count).to be > 1000
```

Comment:
I'd like this to be a specific number. We're working with known files, so we should be getting the same answer here on each run, right? Let's put the specific number in so we don't get off-by-one errors creeping in anywhere.